automated-comments/javascript/general/parse_error.md

```diff
@@ -0,0 +1,7 @@
+Something went wrong whilst parsing the submission; most likely a Syntax Error:
```

Comment:
"whilst" seems to be archaic as of 1913, prefer "while" instead, perhaps?

Reply:
"Syntax Error" quoted, since it goes against standard usage of lower case letters in a sentence?

Reply:
~archaic~ British.

I guess I'll change it.

Reply:
It may be cultural, I usually check against wordnet and the Collaborative International Dictionary of English.

Reply:
```suggestion
Something went wrong while parsing the submission; most likely a "Syntax Error":
```